CHANGES.rst

```diff
@@ -1,10 +1,17 @@
 Changes
 =======
 
-1.4.1 (unreleased)
+2.0.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Fix logic bug in seconds_difference() that could introduce error up to nearly
+  a whole second for any particular event.
+
+- Limit number precision in HTML reports to 3 decimal digits.
+
+- Drop Python 2.6 support.
+
+- Add Python 3.4 through 3.7 support.
```

Comment:
Should we mention the added PyPy support, too?

Reply:
A good question!  I don't know.

I seem to default to mentioning things like that when actual code changes turn out to be necessary to add the support.  When it's just running the tests and seeing them pass, well, then we could say we had pypy support already, it wasn't added now.

But on the other hand it may be useful to note that we will run tests on PyPy continuously and so adding support is like making a promise to keep things working in the future?  But that's what PyPI classifiers state (and .travis.yml), and I'm not sure it's useful to repeat that in the changelog?

Reply:
The PyPy classifier was added in this PR. I'd expect that this change is mentioned in the change log together with the other changes made in code and metadata, so it is not necessary to diff the releases by hand to find the differences.

Sadly there are many projects out there in the world which wo not have a change log at all leaving the work to the developer resp. user to find out what has changed. Having a change log feels a bit like a promise to fill it with all the changes in the project.